contracts/migration/SushiMigrator.sol

```diff
@@ -0,0 +1,189 @@
+// SPDX-License-Identifier: Unlicense
+pragma solidity 0.8.10;
+
+import "../libraries/SafeERC20.sol";
+import "../interfaces/ITreasury.sol";
+import "../interfaces/IUniswapV2Pair.sol";
+import "../interfaces/IUniswapV2Router.sol";
+import "../types/OlympusAccessControlledV2.sol";
+
+/// @notice This contract handles migrating lps from sushiswap to uniswap v2
+///         to make use of this contract ensure contract address is RESERVEMANAGER IN OHM Treasury
+contract SushiMigrator is OlympusAccessControlledV2 {
+    using SafeERC20 for IERC20;
+
+    struct TxDetails {
```

Comment:
Also since this has to be read later the naming should be important here. A/B intermixed with the other is not explicit.  I would also change order. This would be good I think:
```
struct Amounts {
uint128 sushiLpBeforeMigration;
uint128 leftoverSushiLpAfterMigration;
uint128 uniPoolToken0AddedToPool;
uint128 uniPoolToken1AddedToPool;
uint128 uniPoolLpReceived;
uint128 uniPoolToken0ReturnedToTreasury;
uint256 uniPoolToken1ReturnedToTreasury;
}
```

Check below mapping.